spec/shared/workflow_base_shared.rb

```diff
@@ -0,0 +1,25 @@
+shared_examples_for "WorkflowBase" do
```

Comment:
please rename file to shared/workflow_base

Reply:
:+1: done